### DISCUSSION TITLE
Commands fail on remote, while when loggin in via SSH just work...

### DISCUSSION THREAD

**Discussion by hansipete** [deployphp/deployer#3483] [Help needed]
<img width="585" alt="image" src="https://user-images.githubusercontent.com/1027868/218151497-a76fa9a5-6111-4a22-91a7-190191e341c0.png">
This the very specific case, where my deploy fails when running `php ~/xyz.de/releases/19/please cache:clear` 
I'm logged in to the machine via SSH, and rerun the command there: no problem.

Anyone experienced this before? Thank you!

**Comment by antonmedv** [comment]
Check ssh login/nologin options.

**Comment by hansipete** [comment]
Could you point me to the docs please? I was searching but didn't find information about "nologin".

Somehow... I just tried again, and it was working once. The next moment, it's not working again. I'm confused...

**Reply by antonmedv** [reply]
It's linux ops 101)) Will try to write some more docs.
